dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -11031,7 +11031,7 @@ a.category_higlighted, a.tag_higlighted {
 }
 
 .content-search__show-query-dialog .dijitDialogTitleBar {
-  height: 0px;
+  height: 18px !important;
```

Comment:
Please don't use `!important;` unless is the last resource.

Reply:
dijit dialog adding height dynamically on a height that's why I used important.

Reply:
Maybe adding the `height` inline? We can try @zulqarnainvd

Reply:
dijit dialog adding a title bar dynamically so we don't have access to this element or we can use js for adding height.